admin-manual/installation/installation.rst

```diff
@@ -1128,6 +1128,16 @@ It is possible to spread Archivematica's processing load across several machines
 
 For help, send an email to `Archivematica tech mailing list. <https://groups.google.com/forum/#!forum/archivematica-tech>`_
 
+.. _SSL-support:
+
+Using Archivematica through https
```

Comment:
I think I would write it differently, perhaps: "Configure Archivematica with SSL".

I've searched "using nginx through https": only one result. It doesn't seem very common.